package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Install the latest go-ipfs binary",
   "main": "src/index.js",
   "scripts": {
-    "install": "node src/bin.js",
+    "install": "node src/bin.js $TARGET_VERSION $TARGET_OS",
```

Comment:
What happens on Windows with the command? Does `process.argv` get the params as strings or nothing? If latter, then I think we're fine for now, when the params are undefined, it'll download the current platform and version from the package.

If they do get passed in as strings on windows, then we need to figure out something better.